rust/agents/relayer/src/message_processor.rs

```diff
@@ -244,21 +299,21 @@ impl MessageProcessor {
                                 }
                                 MessageProcessingStatus::Processed => {}
                                 MessageProcessingStatus::Error => {
-                                    warn!(
-                                        destination = self.inbox.local_domain(),
+                                    info!(
+                                        destination = self.inbox_contracts.inbox.local_domain(),
                                         leaf_index = leaf_index,
                                         retries = retries,
                                         retry_queue_length = self.retry_queue.len(),
                                         "Retry of message failed processing"
                                     );
                                     if retries >= self.max_retries {
-                                        error!(
-                                        destination = self.inbox.local_domain(),
-                                        leaf_index = leaf_index,
-                                        retries = retries,
-                                        retry_queue_length = self.retry_queue.len(),
-                                        "Maximum number of retries exceeded for processing message"
-                                    );
+                                        info!(
```

Comment:
I think run-locally.sh specifically greps for 'ERROR' for relayer log ouput, to find sufficiently severe messages that should shown to the test runner, so we may need to update that line if we think these sorts of retrys are useful to surface

Reply:
Because this is pretty much expected behavior I'm inclined to just keep in relayer logs and not necessarily surface?